### ISSUE TITLE
Reserve a static space for typing notifications

### ISSUE THREAD

**Issue by DataBeaver** [mirukana/mirage#42]
It's pretty annoying when it causes the entire chat timeline to move up and down when the notification appears and disappears.

**Comment by mirukana** [comment]
By "reserve a static space", you mean to just have empty space between the last message in the timeline and the composer?

**Comment by DataBeaver** [comment]
Yes, that.  Having the empty space there would prevent the messages from shifting up when the typing notification appears and fills the space.

**Comment by ellietheoracle** [comment]
could this be an option, personally i prefer it the other way .

**Comment by mirukana** [comment]
Yes, it should be an option.
